### PR TITLE
LRQA-74555

### PR DESCRIPTION
**Pull request by weller0246**
https://issues.liferay.com/browse/LRQA-74555

### CODE REVIEW

**Comment by weller0246** [comment]
ci:tes:security

**Comment by weller0246** [comment]
ci:test:security

**Comment by weller0246** [comment]
ci:test:security

**Review by gaborlovasdotcom — COMMENTED** [review]
Please contact Core Infra team, to see if ldapadd exists at CI or not?

**Comment by weller0246** [comment]
ci:test:security:nocompile

**Comment by weller0246** [comment]
ci:test:security:nocompile

**Comment by weller0246** [comment]
ci:test:security:nocompile

**Comment by weller0246** [comment]
ci:test:security:nocompile

**Comment by weller0246** [comment]
ci:test:security:nocompile

**Comment by weller0246** [comment]
ci:test:security:nocompile

**Comment by weller0246** [comment]
ci:test:security:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase [hunk 1]
```diff
@@ -97,6 +97,68 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-151394. LDAP user can be unassigned from groups through ldap. Blocked by LPS-153030."
+	@priority = "5"
+	test CanRemoveUserFromGroupThroughLDAP {
+		property apacheds.enabled = "true";
+		property portal.upstream = "marcell";
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase:104]: So it's not blocked anymore by LPS-153030 ;)

Comment (weller0246) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase:104]: Exactly!

**Pull request closed without merging**